schema/meter_provider.json

```diff
@@ -33,6 +33,12 @@
                 },
                 "exporter": {
                     "$ref": "#/$defs/MetricExporter"
+                },
+                "producers": {
```

Comment:
Let's add this as a property on `MetricReader` instead of PeriodicMetricReader and PullMetricReader.

Reply:
Moved